content/docs/tutorials/alerting_based_on_metrics.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Alerting based on metrics.
+sort_rank: 4
+---
+
+# Alerting based on metrics 
+
+In this tutorial we will create alerts on the `ping_request_count` metric that we instrumented earlier. 
```

Comment:
I would somehow mention the tutorial via the link and potentially add another sentence referencing the tutorial of the instrumented server.

Reply:
This was done through the force-push